### PR TITLE
ADAPT-4280: Theme localist widget

### PR DESCRIPTION
**Pull request by Lyafel**
# Summary
- Localist widget theming

# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. https://deploy-preview-276--stanford-alumni.netlify.app/
3. Verify the theming for localist widget at the bottom

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Comment by sherakama** [comment]
What is the path to review the localist embed?

**Comment by Lyafel** [comment]
@sherakama right, sorry. I added it on the home page on alumni dev but it's not visible on the build, is that something I could fix or how does it work?

**Comment by yvonnetangsu** [comment]
> @sherakama right, sorry. I added it on the home page on alumni dev but it's not visible on the build, is that something I could fix or how does it work?

@Lyafel You're free to publish any pages on the **DEV - SAA Alumni** Storyblok space. Only published pages will be built in the Netlify preview deploys.

**Comment by Lyafel** [comment]
@yvonnetangsu How do make it rebuild? I published the node but the build still doesn;t show the changes

**Comment by yvonnetangsu** [comment]
> @yvonnetangsu How do make it rebuild? I published the node but the build still doesn;t show the changes

@Lyafel You could log into Netlify, click into your branch deploy preview and click "Deploy site" again :)
Screen Shot 2022-01-26 at 2 36 16 PM

**Comment by Lyafel** [comment]
@yvonnetangsu Now I know everything, thanks :D

**Comment by Lyafel** [comment]
@sherakama The block is visible on the build now, front page at the bottom

**Review by yvonnetangsu — CHANGES_REQUESTED** [review]
@Lyafel This is looking really great! A few things:

- Could we change the card headline to serif?
- The month portion of the date in the circle should left align with the day number
- Could we get rid of that dotted line under the month in the circle? And add a bit more space underneath (so it matches the Event Cards)?
- If possible please add the external link icon after the card title

File: src/styles/localist.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  @apply su-stretched-link su-no-underline hocus:su-underline su-underline-offset !su-underline-thick !su-underline-digital-red-xlight su-text-black hocus:su-text-black;
+}
+
+.start-time {
+  @apply su-card-paragraph su-rs-px-2 su-flex su-flex-row su-items-center md:su-items-start su-mb-04em;
+}
+
+.start-time::before {
+  content: "";
+  background-image: url("../asseets/calendar.svg");
```
Comment (yvonnetangsu) [inline src/styles/localist.css:58]: Would be great to fix up the folder name typo "asseets" to "assets". Hmm actually wouldn't that needs to be place inside the /static folder for Gatsby also?

File: src/styles/localist.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
+.event-list {
+  @apply su-grid su-grid-cols-1 md:su-grid-cols-2 xl:su-grid-cols-3 su-justify-items-center su-items-stretch su-gap-y-xl md:su-gap-y-[5rem] xl:su-gap-y-[7rem] su-grid-gap su-p-0;
+}
+
+.event {
+  @apply su-flex su-flex-col su-relative su-overflow-hidden sm:su-max-w-[42rem] md:su-max-w-full su-text-black su-break-words su-w-full su-rs-pb-3 su-bg-white su-border su-border-solid su-bg-clip-padding su-shadow-sm focus-within:su-shadow-md hover:su-shadow-md su-backface-hidden su-border-black-30-opacity-40;
+  font-size: 2.1rem;
```
Comment (yvonnetangsu) [inline src/styles/localist.css:7]: How come not using su-text-21 here? 🤔

**Comment by yvonnetangsu** [comment]
@Lyafel Sorry miss a few 😂 
- Please add an object-fit:cover (TW equiv) to the image so it's not distorted
- Looks like when one hovers over the date circle portion of the card, it becomes not clickable - maybe a z-index issue?

File: src/styles/localist.css (hunk 1 above)
Comment (Lyafel) [reply src/styles/localist.css]: I looked a bit at it, looks like it's recommended to call assets from js not from static for simpler cases, but also I didn't manage to call the icons from static to css.

**Comment by Lyafel** [comment]
Done thought couldn't align the dates correctly because of the structure available.

**Review by yvonnetangsu — CHANGES_REQUESTED** [review]
@Lyafel This looks fantastic! If you could please left align the day and month inside the circle, then this is GTG. Thanks!
Screen Shot 2022-02-02 at 12 20 36 PM

**Comment by yvonnetangsu** [comment]
> Done thought couldn't align the dates correctly because of the structure available.

Oh - sorry just saw this comment. Let me take a look

**Comment by yvonnetangsu** [comment]
@sherakama I pushed up some final tweaks (responsive font sizes, icons, date circle) - final approval is yours 😄

**Comment by sherakama** [comment]
@yvonnetangsu 
Is the word 'sponsors' supposed to be there?

https://share.getcloudapp.com/eDulrGk9

**Comment by yvonnetangsu** [comment]
> @yvonnetangsu Is the word 'sponsors' supposed to be there?
> 
> https://share.getcloudapp.com/eDulrGk9

Yeah, the clients added it on the production sites under the ads. If we're doing this for real though, the whole localist section will need to be in its own section and not inside the same section as the other ads, so this should be ok

**Review by sherakama — APPROVED** [review]
GTG. Thank you.

**Pull request merged**